routes/flag.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+const authenticate = require("../middlewares/authenticate");
+const { authorizeUser } = require("../middlewares/authorization");
+const flagController = require("../controllers/flag");
+const { addFeatureFlag } = require("../middlewares/validators/flag");
+const { SUPER_USER } = require("../constants/roles");
+
+router.post("/add", authenticate, authorizeUser(SUPER_USER), addFeatureFlag, flagController.addFlag);
```

Comment:
I find the name of middleware`addFeatureFlag` a little misleading. Should it renamed to demonstrate that it validates the incoming request?

Reply:
Like, maybe  'validateAddingFeatureFlagReq`?

Reply:
So i named this function as `addFeatureFlagValidator` earlier but changed to `addFeatureFlag` based upon a review. Is that name good enough or need to write some different name.

Reply:
How about importing it as 
```js
const validator = require("../middlewares/validators/flag");
```
So it becomes `validator.addFeatureFlag`?